lisa/schema.py

```diff
@@ -1586,6 +1586,9 @@ class TestCase(BaseTestCaseFilter):
     # it uses to work around some cases temporarily, don't overuse it.
     # default is false
     ignore_failure: bool = False
+    # skip_hyperv_cases means skip all hyper-v related test cases
+    # when it is true. Default is false.
+    skip_hyperv_cases: bool = False
```

Comment:
Implement it as a feature, called `Virtualization` with a field called `host_type` (add comments to make the name clearly). The default value should be None, and no limits. It could be BareMetal, HyperV, cloud hypervisor, qemu and so on.

It should be always added to HyperV and Azure platforms.

Reply:
updated